src/frontend/src/pages/HomePage.jsx

```diff
@@ -1,54 +1,22 @@
-import { useState } from 'react';
-import { useAuth } from '@/hooks/useAuth';
-import Container from 'react-bootstrap/Container';
-import { getUserFromSession } from '@/utils/getUserFromSession';
+import { Container, Col, Row } from 'react-bootstrap';
+import SearchForm from '@/components/SearchForm/SearchForm';
```

Comment:
Please import  like this: 

import {SearchForm, BackgroundContainer} from '@/components';

and the SearchForm and BackgroundContainer should be set in the index.js of course